src/downloadlistview.cpp

```diff
@@ -34,8 +34,9 @@ along with Mod Organizer.  If not, see <http://www.gnu.org/licenses/>.
 using namespace MOBase;
 
 DownloadProgressDelegate::DownloadProgressDelegate(DownloadManager* manager,
-                                                   DownloadListView* list)
-    : QStyledItemDelegate(list), m_Manager(manager), m_List(list)
+                                                   DownloadListView* list,
+                                                   DownloadList* sourceModel)
+    : QStyledItemDelegate(list), m_Manager(manager), m_List(list), m_sourceModel(sourceModel)
```

Comment:
More of a remark, but having to pass the `DownloadList` only to do one `getDownloadByRow` seems a bit overkill.

Reply:
this is now being used to pull the fileName from the current row in order to get the downloadListItem